lib/rubocop.rb

```diff
@@ -592,6 +592,7 @@
 require_relative 'rubocop/cop/rails/present'
 require_relative 'rubocop/cop/rails/read_write_attribute'
 require_relative 'rubocop/cop/rails/redundant_receiver_in_with_options'
+require_relative 'rubocop/cop/rails/reflection_class_name_string'
```

Comment:
Thank you for renewing the cop name. Please change the file name according to cop class name.

```diff
-require_relative 'rubocop/cop/rails/reflection_class_name_string'
+require_relative 'rubocop/cop/rails/reflection_class_name'
```